generators/entity-server/templates/src/test/java/package/web/rest/EntityResourceIT.java.ejs

```diff
@@ -1043,18 +1043,6 @@ _%>
         default<%= entityClass %>ShouldNotBeFound("<%= searchBy.fieldName %>.equals=" + <%= 'UPDATED_' + searchBy.fieldNameUnderscored.toUpperCase() %>);
     }
 
-    @Test<%= transactionalAnnotation %>
```

Comment:
Doesn't make sense semantically.
Also, now faker & liquibase are imported at test start, which makes it fail (but brings a lot more possibility in term of integration tests)